<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SIMS</class>
 <widget class="QMainWindow" name="SIMS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SIMS</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>701</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="StudUsrnLabel">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>90</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QLabel" name="StudPwdLabel">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>150</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="StudUsrnLE">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>90</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="StudPwdLE">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>150</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="StudLoginBtn">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>220</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>login</string>
      </property>
     </widget>
     <widget class="QLabel" name="welcomeLabel">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>9</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Welcome to the Login page</string>
      </property>
     </widget>
     <widget class="QLabel" name="StudLoginStatusLabel">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>120</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>logout</string>
      </property>
     </widget>
     <widget class="QPushButton" name="StudInfoButn">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>300</y>
        <width>371</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>View info</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>110</y>
        <width>301</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>My grades</string>
      </property>
      <widget class="QLabel" name="Test1Label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>47</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Test 1</string>
       </property>
      </widget>
      <widget class="QLabel" name="Test1Grade">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>20</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>N/A</string>
       </property>
      </widget>
      <widget class="QLabel" name="Test2Label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Test 2</string>
       </property>
      </widget>
      <widget class="QLabel" name="Test2Grade">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>50</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>N/A</string>
       </property>
      </widget>
      <widget class="QLabel" name="Test3Label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Test 3</string>
       </property>
      </widget>
      <widget class="QLabel" name="Test3Grade">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>80</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>N/A</string>
       </property>
      </widget>
      <widget class="QLabel" name="AverageLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Average</string>
       </property>
      </widget>
      <widget class="QLabel" name="AverageGrade">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>110</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>N/A</string>
       </property>
      </widget>
      <widget class="QLabel" name="StudGPAlabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>GPA</string>
       </property>
      </widget>
      <widget class="QLabel" name="StudGPAvalue">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>140</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>N/A</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="TheStudentName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>331</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Student name</string>
      </property>
     </widget>
     <widget class="QLabel" name="TheStudentID">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="ProfessorName">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Select a course.</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="InfoBackButn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>361</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>My Information</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>First Name:</string>
       </property>
      </widget>
      <widget class="QLabel" name="StudFNameLabel">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>50</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Last Name:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>110</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>140</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Username:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>170</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
      <widget class="QLabel" name="StudLNameLabel">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>80</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="StudIDlabel">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>110</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="StudInfoUsernameLabel">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>140</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="StudInfoPasswordLabel">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>170</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>735</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
